spec/directions/bake_chapter_glossary_spec.rb

```diff
@@ -5,8 +5,82 @@
   # TODO: test that...
   # 1) definitions with the same term sort next by the definition
 
-  xit 'works' do
+  let(:doc) do
+    Kitchen::BookDocument.new(document: Nokogiri::XML(
+      <<~HTML
+        <html>
+        <body>
+          <div data-type="chapter">
+            <div data-type='glossary'>
+              <div>
+                <dl>
+                  <dt>ZzZ</dt>
+                  <dd>Test 1</dd>
+                </dl>
+                <dl>
+                  <dt>ZzZ</dt>
+                  <dd>Achoo</dd>
+                </dl>
+                <dl>
+                  <dt>ABD</dt>
+                  <dd>Test 2</dd>
+                </dl>
+              </div>
+            </div>
+          </div>
+        </body>
+        </html>
+      HTML
+    ))
+  end
```

Comment:
Add a `chapter_element(chapter_children_html)` helper method to https://github.com/openstax/kitchen/blob/master/spec/helpers/factory_helpers.rb like [this one for page_element](https://github.com/openstax/kitchen/pull/89#discussion_r564746908).  Then this can be `let(:chapter) ...`